packages/babel-preset/src/index.js

```diff
@@ -89,8 +89,8 @@ const plugin = (api, opts) => {
     plugins.push(svgDynamicTitle)
   }
 
-  if (opts.native) {
-    plugins.push(transformReactNativeSVG)
+  if (opts.native || opts.expo) {
+    plugins.push([transformReactNativeSVG, { expo: !!opts.expo }])
```

Comment:
Hmm, I think we should use `native: { expo: true }` instead of a direct `expo` option.